azext_connect/custom.py

```diff
@@ -556,3 +556,21 @@ def validate_general(cmd, resource_group, name):
         print(e)
         logger.error(e)
         sys.exit(1)
+
+
+def get_general(cmd, resource_group, name):
+    try:
+        subscription = get_subscription_id(cmd.cli_ctx)
+        api = _create_api(cmd)
+        result = api.get(subscription, resource_group, name)
+        if result.ok is not True:
+            end = result.text.find('\r\n\r\nHEADERS\r\n=======')
```

Comment:
Is this a workaround?

Reply:
Here i follow the structure in validate and create, and here can be modified if necessary.

Reply:
I would suggest refining this if we have bandwidth.